### PR TITLE
Use MutliSelect for dropdown questions

### LINKED ISSUE #498: Use the multiselect component in dropdown component
> Because the call on whether to use the multiselect component or its own component has changed around so much, I’d prefer if you can create that follow-up issue with the relevant info. :)

That would make the create preview also be the same as what the real final form would look like :rocket:

cc @jancborchardt
_Originally posted by @skjnldsv in https://github.com/nextcloud/forms/pull/461#pullrequestreview-443725230_

### PR DESCRIPTION
**Pull request by Chartman123**
This fixes #498 

Signed-off-by: Christian Hartmann <chris-hartmann@gmx.de>

### CODE REVIEW

**Comment by Chartman123** [comment]
@jotoeri Do you know why the selected option doesn't show up in the dropdown? If I understand it correctly that should be the default behaviour for the `option` slot...

**Comment by jotoeri** [comment]
Maybe you need to provide a variable for storing this. Thus having a variable and using v-model should work, i think :thinking: 
https://nextcloud-vue-components.netlify.app/#/Components/Multiselect

**Comment by Chartman123** [comment]
> Thus having a variable and using v-model should work

@jotoeri Thanks, that did the trick :)

**Comment by jancborchardt** [comment]
Ah I get it, this is only for the creation process?

Considering your question in the issue @Chartman123 
> Do you mean that the component should be used to add new options, too, or only for the "real" preview as long as you're not in the edit mode for the options? Because I couldn't find a way to add new options directly within the Multiselect component other than for tags.

Ideally yes, it could be possible to add new options there too. But @skjnldsv or other Vue component experts will have to chime in on if that’s already possible.

**Comment by Chartman123** [comment]
> Or is this only to use the component and multiselection is limited?

By now the dropdown allows only one option to be selected, however, there are some code elements prepared for multiple options to be selected.

Yes would be nice if some Vue experts could join this conversation :)

File: src/components/Questions/QuestionDropdown.vue [hunk 1]
```diff
@@ -36,23 +36,16 @@
...
-				{{ selectOptionPlaceholder }}
-			</option>
-			<option v-for="answer in options"
-				:key="answer.id"
-				:value="answer.id"
-				:selected="isChecked(answer.id)">
-				{{ answer.text }}
-			</option>
-		</select>
+			:options="options"
```
Comment (jotoeri) [inline src/components/Questions/QuestionDropdown.vue:44]: Placeholder is missing 😉

**Review by jotoeri — COMMENTED** [review]
Just one small thing.

**Comment by jotoeri** [comment]
> Ideally yes, it could be possible to add new options there too. 

So as far as i see, the MultiSelect Component does currently not provide that functionality.

> Or is this only to use the component and multiselection is limited?

Yes, i'd keep this one for now only to move the dropdown to the component. We can add the other functionality later.

And here's a screenshot of how users can submit then: 😉 
grafik

**Review by jancborchardt — APPROVED** [review]
Seems good to me then :)

File: src/components/Questions/QuestionDropdown.vue (hunk 1 above)
Comment (Chartman123) [reply src/components/Questions/QuestionDropdown.vue:45]: Fixed it, so we're good to go 🙂

**Pull request merged**